src/index.html

```diff
@@ -11,6 +11,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="forms">
+      <div class="form-big">
+        <form
+          action="#"
+          method="get"
+          data-qa="big"
+          class="searchbar searchbar--big"
+        >
+          <label class="label searchbar">
```

Comment:
Remove this `label` tag. You don't need it because you haven't any text as a label for input.